prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -308,5 +312,157 @@ class HttpUtilities {
                 return responseCode to response
             }
         }
+
```

Comment:
We had talked about reusing the generic `put, get, delete`, etc, methods already written in `cli/SettingsCommands.kt`.

I think all of this code can be removed, and replaced with calls into those methods.  You may need to pull those methods into the `companion object `in order to use them without having to instantiate a SettingsCommand object.